addons/base_import/tests/test_base_import.py

```diff
@@ -250,23 +259,27 @@ def test_success(self):
         # Ensure we only have the response fields we expect
         self.assertItemsEqual(result.keys(), ['matches', 'headers', 'fields', 'preview'])
 
-class test_convert_import_data(TransactionCase):
+
+class TestConvertImportData(TransactionCase):
     """ Tests conversion of base_import.import input into data which
     can be fed to Model.import_data
     """
+    def setUp(self):
+        """*****setUp*****"""
+        super(TestConvertImportData, self).setUp()
+        self.BaseImport = self.env['base_import.import']
```

Comment:
`self.env['base_import.import']` used more than one time in this , so i create instance alias which can used at multiple place.